source/includes/api/_areas_v2.md

```diff
@@ -0,0 +1,280 @@
+-H "Authorization: Bearer <your-token>" \
+-H "Content-Type: application/json" -d \
+ '{
+    "name": "Example area",
+    "application": "gfw",
+    "image": "",
+    "tags": [],
+    "public": false,
+    "fireAlerts": true,
+    "deforestationAlerts": true,
+    "webhookUrl": "http://example.com",
+    "monthlySummary": true,
+    "email": "youremail@resourcewatch.org",
+    "language": "en"
+}'
+```
+
+> Example response:
+
+```json
+{
+    "data": {
+        "type": "area",
+        "id": "5e4d7c47ef240412c2d56a78",
+        "attributes": {
+            "name": "Example area",
+            "application": "gfw",
+            "wdpaid": null,
+            "userId": "5e2f0eaf9de40a6c87dd9b7d",
+            "createdAt": "2020-02-19T18:19:51.485Z",
+            "image": "",
+            "datasets": [],
+            "tags": [],
+            "status": "pending",
+            "public": false,
+            "fireAlerts": true,
+            "deforestationAlerts": true,
+            "webhookUrl": "http://example.com",
+            "monthlySummary": true,
+            "subscriptionId": "5e4d7c47dd8fa31290d548ae",
+            "email": "youremail@resourcewatch.org",
+            "language": "en"
+        }
+    }
+}
+```
+
+Use this endpoint to update an existing area. This endpoint requires authentication.
+
+This endpoint supports the same request body parameters as the `POST area` endpoint.
+
+**Implementation details**
+
+PATCHing an area is a bit more complex, and it comes down to 3 major cases:
+
+1. The area already exists and has subscriptions preference (`deforestationAlerts`, `fireAlerts` or `monthlySummary` set to true) in the request data:
+   1. If the area doesn't have a subscription associated, a new one is created and associated.
+   2. If the area already had a subscription, then the subscription is PATCHed according to the data provided in the request body.
+2. The area already exists and doesn’t has subscription preferences (`deforestationAlerts`, `fireAlerts` or `monthlySummary` set to true) in the request data:
+   1. If the area had a subscription associated, then the subscription associated is deleted.
+   2. Otherwise, just save the area.
+3. The area doesn’t exist because on the fetch it returned a mapped subscription (meaning we are PATCHing an area using the ID of a subscription):
+   1. First, create a new area, and then:
+      1. If the request data has subscriptions preference (`deforestationAlerts`, `fireAlerts` or `monthlySummary` set to true), also PATCH the subscription.
+      2. If the request data doesn't have subscriptions preference (`deforestationAlerts`, `fireAlerts` or `monthlySummary` set to true), delete the associated subscription.
+
+## Deleting an area
+
+```shell
+curl -X DELETE https://api.resourcewatch.org/v2/area/:id
+-H "Authorization: Bearer <your-token>" \
+```
+
+> Returns 204 No Content in case of success.
+
+Use this endpoint to delete an existing area. This endpoint requires authentication.
```

Comment:
Add more details on who can delete an area - seems to be limited to the owner of the area or ADMIN users